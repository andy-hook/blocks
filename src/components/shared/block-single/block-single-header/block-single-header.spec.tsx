import React from "react"
import BlockSingleHeader from "./block-single-header"
import { renderWithTheme } from "@test-utils"

describe("<BlockSingleHeader />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <BlockSingleHeader />)
    expect(tree).toMatchSnapshot()
  })
})
