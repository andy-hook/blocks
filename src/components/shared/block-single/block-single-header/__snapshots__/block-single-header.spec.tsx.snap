// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockSingleHeader /> renders correctly 1`] = `
<div
  className="sc-gzVnrw iCGcXq"
>
  <header
    className="sc-htoDjs dEgfPh"
  >
    <div
      className="sc-iwsKbI ldERec"
    >
      <div
        className="sc-htpNat iflsxI sc-dnqmqq dpESLn"
        size="lg"
      >
         Transactions
      </div>
      <div
        className="sc-bdVaJa jlBogh"
        size="lg"
      >
        #Â 
      </div>
    </div>
    <section
      className="sc-ifAKCX kWjkTO"
    >
      <button
        className="sc-EHOje jvzENE"
      >
        Details
      </button>
      <button
        className="sc-bZQynM bOrPFB"
      >
        Transactions
      </button>
    </section>
  </header>
</div>
`;
